### ISSUE TITLE
Use of `serialized_attributes` is deprecated and will be removed in Rails 5.0

### ISSUE THREAD

**Issue by unchris** [Shopify/identity_cache#202]
Our codebase migrated to Rails 4.2 recently and we are seeing a lot of deprecation warnings. One of them appears to be getting fired from [Identity Cache](https://github.com/Shopify/identity_cache/blob/483b5f24880abde790c2ef091e51c23bbc09543d/lib/identity_cache/query_api.rb#L83):

```
DEPRECATION WARNING: `serialized_attributes` is deprecated without replacement, and will be removed in Rails 5.0
```

**Comment by unchris** [comment]
https://github.com/rails/rails/pull/15704

**Comment by dylanahsmith** [comment]
Fixed by pull request #208
